lib/new_relic/agent/instrumentation/grpc/client/instrumentation.rb

```diff
@@ -13,26 +13,48 @@ module GRPC
         module Client
           include NewRelic::Agent::Instrumentation::GRPC::Helper
 
-          def issue_request_with_tracing(method, requests, marshal, unmarshal,
+          def issue_request_with_tracing(grpc_type, method, requests, marshal, unmarshal,
```

Comment:
- Have each instrumented client request sending method report its name to
  `issue_request_with_tracing`, and then report that name was `grpc_type`